apps/client/src/setupTests.ts

```diff
@@ -9,9 +9,9 @@ window.matchMedia =
   window.matchMedia ||
   function () {
     return {
-      matches: false,
       // eslint-disable-next-line @typescript-eslint/no-empty-function
       addListener: function () {},
+      matches: false,
```

Comment:
Was hot des mit eslint zum tua. Besser gfrogt was macht des?

Reply:
The code itself nothing. But it failed the linter so I had to make sure to fix it.